gui/default/syncthing/core/syncthingController.js

```diff
@@ -62,7 +62,7 @@ angular.module('syncthing.core')
                 selectedDevices: {},
                 type: "readwrite",
                 rescanIntervalS: 60,
-                minDiskFreePct: 1,
+                minDiskFree: "1 %",
```

Comment:
In the folder edit modal this is represented as a struct with value and unit members, might be the culprit for the space units Audrius mentioned.